app/models/legal_aid_application.rb

```diff
@@ -194,8 +194,8 @@ def add_benefit_check_result
 
     self.benefit_check_result ||= build_benefit_check_result
     benefit_check_result.update!(
-      result: benefit_check_response.dig(:benefit_checker_status),
-      dwp_ref: benefit_check_response.dig(:confirmation_ref)
+      result: benefit_check_response[:benefit_checker_status],
+      dwp_ref: benefit_check_response[:confirmation_ref]
```

Comment:
I thought `dig` was a safe navigation operator, why has rubocop invalidated it?!